packages/ledger.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Ledger < Package
+  description 'A double-entry accounting system with a command-line reporting interface'
+  homepage 'https://www.ledger-cli.org/'
+  version '3.1.3'
+  source_url 'https://github.com/ledger/ledger/archive/v3.1.3.tar.gz'
+  source_sha256 'b248c91d65c7a101b9d6226025f2b4bf3dabe94c0c49ab6d51ce84a22a39622b'
+
+  binary_url ({
+    aarch64: '',
+     armv7l: '',
+       i686: '',
+     x86_64: '',
+  })
+  binary_sha256 ({
+    aarch64: '',
+     armv7l: '',
+       i686: '',
+     x86_64: '',
+  })
+
+  depends_on 'gmp' => :build
+  depends_on 'boost' => :build
+  depends_on 'mpfr' => :build
```

Comment:
All of these dependencies come along for the ride in Chromebrew "core", so they aren't needed here.

Reply:
Actually, `boost` is not in "core" so that line is fine.